content/en/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -286,11 +286,11 @@ spec:
         host: httpbin
         subset: v1
       weight: 100
-    mirror:
-      host: httpbin
-      subset: v2
-    mirrorPercentage:
-      value: 100.0
+      mirror:
+        host: httpbin
+        subset: v2
+      mirrorPercentage:
+        value: 100.0
```

Comment:
This change is not correct. The `mirror` field is part of HTTPRoute struct as currently documented.
https://istio.io/latest/docs/reference/config/networking/virtual-service/#HTTPRoute